starter-code/lib/lodash.js

```diff
@@ -0,0 +1,40 @@
+//var _ = require("lodash");
+
+var opinions = [ "This is the best job ever",
+                 "Satisfied",
+                 "At least I get paid",
+                 "I'm looking for another job",
+                 "I don't want to answer"];
+
+var departments = [];
+var selectedOpinions = [];
+
+departments = _.times(5, function() {
+  var tempArray1 = [];
+  var tempArray2 = [];
+  selectedOpinions = _.times(10, function(){
+  var randomNumber = Math.floor(Math.random() * opinions.length);
+  tempArray2.push(opinions[randomNumber]);
+  return tempArray2;
+  });
+
+  tempArray1.push(tempArray2);
+  return tempArray2;
+});
+
```

Comment:
The overall result is good but you should try to divide each process in different functions. 
i.e. a function to generate a random opinion, then a function to populate an array with 10 random opinions and then another one to populate the 5 departments, so if you need to reuse those functions again, you can do it.